### DISCUSSION TITLE
Obscure error from antismash

### DISCUSSION THREAD

**Discussion by Bio-finder** [antismash/antismash#677] [Q&A] [answered]
Hello,
I have just a question concerning an error that I received and that I don't understand:
`ERROR    24/11 10:10:55   all records skipped`
What is wrong with my records? I checked the size (over 1000 bp) for at least one record before to submit it so it's not the size which is the problem.

**Comment by SJShaw** [comment, marked as answer]
There's a few reasons a record might be skipped, some of them much less likely than others:
- too small, as you've already checked
- no CDS features are annotated in the input and `--genefinding-tool` is set as `ignore` (the default on the webservice for GBK inputs) or no CDS features are found with the chosen method
- no protoclusters are detected
- `--limit N` is set and records that wouldn't otherwise be skipped are the Nth record or later
- all CDS features present are over 100kb

If you run with `--debug`, there'll be messages that will give you an indicator as to what's going on. They look like this:
> Not annotating skipped record \<record id>: \<reason>

**Reply by Bio-finder** [reply]
Thank you. This might be the file that I submit as I send a fasta but I work on the offline tool.

**Reply by Bio-finder** [reply]
I found just that no protoclusters are detected in my samples but for me it shouldn't send an error, just return an empty file and send a warning as there is no issue in the way the software worked, it's just that nothing is detected. This can happen.
